plugins/datetime/Makefile.am

```diff
@@ -5,16 +5,17 @@ polkitdir = $(datadir)/polkit-1/actions
 dbus_services_in_files = org.mate.SettingsDaemon.DateTimeMechanism.service.in
 polkit_in_files = org.mate.settingsdaemon.datetimemechanism.policy.in
 
-msd-datetime-mechanism-glue.h: $(srcdir)/msd-datetime-mechanism.xml
-	$(AM_V_GEN) dbus-binding-tool \
-			--prefix=msd_datetime_mechanism --mode=glib-server	\
-			--output=msd-datetime-mechanism-glue.h			\
-			$(srcdir)/msd-datetime-mechanism.xml
+$(BUILT_SOURCES) : $(srcdir)/msd-datetime-mechanism.xml
+	$(AM_V_GEN) $(GDBUS_CODEGEN) \
+		--interface-prefix org.mate.SettingsDaemon. \
+		--c-namespace MateSettings \
+		--generate-c-code msd-datetime-generated $<
```

Comment:
Previous warning is fixed. But a similar warning is in `msd-datetime-generated.c`
Sorry, i didn't saw that in last review.
https://app.travis-ci.com/github/mate-desktop/mate-settings-daemon/jobs/610161263#L4693
```
msd-datetime-generated.c:1821:18: warning: declaration shadows a variable in the global scope [-Wshadow]

    const gchar *timezone)

   /usr/include/time.h:233:17: note: previous declaration is here

extern long int timezone;

1 warning generated.
```
I do not not see from which source file the code will generated and the *generated build sources will be used for polkit.

Reply:
`gdbus-codegen` will generate this files. So it seems the warning can't be fixed.

Reply:
Because we use the "timezone" string parameter in the xml file, so this code is generated.

```
<arg name="timezone" direction="out" type="s"/>
```